src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -175,6 +175,13 @@
 
     <ArcadeSharedFrameworkSdkOverride Include="Microsoft.DotNet.SharedFramework.Sdk" Group="ARCADE_SHARED_FX_SDK" Version="$(arcadeOutputPackageVersion)" />
     <WindowsDesktopSdkOverride Include="Microsoft.Net.Sdk.WindowsDesktop" Group="WINDOWS_DESKTOP" Location="$(ToolsDir)EmptySdk" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
+    <!-- Discover all versions of Microsoft.Build.Traversal and Microsoft.Build.NoTargets built in SBRP. -->
```

Comment:
I see the Traversal and the NoTargets msbuild sdks similar to the Arcade SDK. We only use a single version of the live built Arcade SDK (aside from the bootstrap one). Why can't we have the same for NoTargets and Traversal and upgrade to a single version? IMO that makes more sense.

Reply:
I did attempt that and hit issues in the build. I'll do another test.

Reply:
Feel free to send me the error on Teams. Repositories might need small adjustments but using a consistent version for these msbuild sdks seems highly valuable to me.

Reply:
Worked fine now that `razor` and `sdk` projects do not require explicit SDK versions.

Fixed with https://github.com/dotnet/installer/pull/18478/commits/1e17ce56ede64048cd8cc068238e24ab32662045